plugins/win-capture/window-capture.c

```diff
@@ -234,6 +288,29 @@ static void wc_tick(void *data, float seconds)
 		dc_capture_free(&wc->capture);
 		dc_capture_init(&wc->capture, 0, 0, rect.right, rect.bottom,
 				wc->cursor, wc->compatibility);
+
+		struct dstr title = {0};
+		struct dstr class = {0};
+		struct dstr exec = {0};
+		struct dstr win = {0};
+
+		get_window_exe(&exec, wc->window);
+		get_window_class(&class, wc->window);
+		get_window_title(&title, wc->window);
+
+		dstr_printf(&win, "%s:%s:%s", title.array, class.array,
```

Comment:
This will likely break for window titles containing `:`; see `encode_dstr` https://github.com/obsproject/obs-studio/blob/49fceca0fe40fa49704aee358f6072f1b8964c40/plugins/win-capture/window-helpers.c#L10-L23 on an escaping scheme that seems to work? You probably also want to unescape in your modified callback